plugin/admin/options/class-options.php

```diff
@@ -106,12 +106,12 @@ public static function init() {
 
 		self::$root_path = new \Sgdd\Admin\Options\OptionTypes\PathField( 'root_path', '', 'basic', 'path_selection', [] );
 
-		self::$embed_width  = new \Sgdd\Admin\Options\OptionTypes\IntegerField( 'embed_width', __( 'Width', 'skaut-google-drive-documents' ), 'advanced', 'file', '800' );
-		self::$embed_height = new \Sgdd\Admin\Options\OptionTypes\IntegerField( 'embed_height', __( 'Height', 'skaut-google-drive-documents' ), 'advanced', 'file', '600' );
+		self::$embed_width  = new \Sgdd\Admin\Options\OptionTypes\StringField( 'embed_width', __( 'Width', 'skaut-google-drive-documents' ), 'advanced', 'file', '100%' );
+		self::$embed_height = new \Sgdd\Admin\Options\OptionTypes\StringField( 'embed_height', __( 'Height', 'skaut-google-drive-documents' ), 'advanced', 'file', '600' );
```

Comment:
Could we have the default `600px` please in order for it to be better understandable?

Reply:
100% for nearly any situation will be proper value - embedded document just fill parent container. In general width expressed in pixels is against responsive design (usually it is a big problem for mobile devices which have much smaller screens than regular computers). In present time webpage should render on any screen size properly.

Reply:
Oh, I meant the height :)

Reply:
Ok - I changed default height from 600 to 600px.